packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -227,7 +216,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           onDismiss={ this._onDismiss }
           onPositioned={ this._onPositioned }
         >
-          <div style={ { width: this._dropDown.clientWidth - 2 } }>
+          <div style={ { width: this._dropDown.clientWidth - 2, maxHeight: calculatedHeight === 0 ? undefined : calculatedHeight } }>
```

Comment:
If this is the main point of the PR, why not just have users pass in the `maxHeight`?